### PR TITLE
Include more updates to NEWS.md and AUTHORS for 1.7

### PR DESCRIPTION
**Pull request by itchyny**
This will be merged just before the release so still in draft. Maintainers, please comment below if you find some other notable changes.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -7,16 +7,16 @@ Since the last stable release many things have happened:
 - jq now lives at <https://github.com/jqlang>
 - New maintainers, admins, and owners have been recruited.
   - A list of [current maintainers](https://github.com/jqlang/jq/blob/70bbd10b0b58e797d03963264fc934879bb44454/AUTHORS#L4-L13)
-- CI, scan builds, release builds etc now use GitHub actions. @owenthereal #2596 #2620
+- CI, scan builds, release, website builds etc now use GitHub actions. @owenthereal @itchyny #2596 #2603 #2620 #2723
```
Comment (wader) [inline NEWS.md]: Can probably add my name to this also :)

**Comment by wader** [comment]
A bit meta but could add note about NEWS.md

File: NEWS.md (hunk 1 above)
Comment (itchyny) [reply NEWS.md]: Of course!

File: NEWS.md [hunk 2]
```diff
@@ -6,17 +6,17 @@ Since the last stable release many things have happened:
...
+- CI, scan builds, release, website builds etc now use GitHub actions. @owenthereal @wader @itchyny #2596 #2603 #2620 #2723
 - Lots of documentation improvements and fixes.
-- Web site updated with new auto complete, better section ids for linking, dark mode, etc. @itchyny #2628
+- Website updated with new section search box, better section ids for linking, dark mode, etc. @itchyny #2628
 - Release builds for:
   - Linux `amd64`, `arm64`, `armel`, `armhf`, `i386`, `mips`, `mips64`, `mips64el`, `mips64r6`, `mips64r6el`, `mipsel`, `mipsr6`, `mipsr6el`, `powerpc`, `ppc64el`, `riscv64` and `s390x`
   - macOS `amd64` and `arm64`
   - Windows `i386` and  `amd64`
   - Docker `linux/386`, `linux/amd64`, `linux/arm64`, `linux/mips64le`, `linux/ppc64le`, `linux/riscv64` and `linux/s390x`
   - More details see @owenthereal #2665
```
Comment (wader) [inline NEWS.md:19]: Mention https://github.com/jqlang/jq/pull/2801 somewhere around here?

**Comment by owenthereal** [comment]
LGTM.

The [ChangeLog](https://github.com/jqlang/jq/blob/master/ChangeLog) can be removed now?

**Comment by nicowilliams** [comment]
We should add @mrwilson for #2837.

**Comment by nicowilliams** [comment]
Any objections to merging this and doing a `1.7rc2` release?

**Comment by nicowilliams** [comment]
Well, let's merge this now anyays.  We can add any other missing news and authors after rc2.

**Comment by nicowilliams** [comment]
Thanks!

**Pull request merged**